### DISCUSSION TITLE
how to update hyprdots?

### DISCUSSION THREAD

**Discussion by melonges** [prasanthrangan/hyprdots#262] [Q&A] [answered]
I am following this project and watching its new commits. How would it be right to pull  the changes to the already installed hyprdots in the system?

**Comment by prasanthrangan** [comment, marked as answer]
Hi, you just have to pull the latest updates and rerun the install script as  -

```
❯ cd ~/Hyprdots/Scripts   # your local clone dir
❯ git pull
❯ ./install.sh -r
```
